ql/src/test/queries/clientpositive/windowing_tinyint.q

```diff
@@ -0,0 +1,11 @@
+--! qt:dataset:src
```

Comment:
Please remove this line since this test doesn't use the `src` table

Reply:
Done